fbw-common/src/systems/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx

```diff
@@ -9,233 +9,254 @@ import { CloudArrowDown } from 'react-bootstrap-icons';
+                </div>
+              </div>
+              <div>
+                <div className="flex w-full flex-row items-center">
+                  <p className={`font-body ${flightPlanProgress > 1 ? 'text-theme-highlight' : 'text-theme-text'}`}>
+                    {schedOutParsed}
+                  </p>
+                  <div className="relative mx-6 flex h-1 w-full flex-row">
+                    <div className="border-theme-text absolute inset-x-0 border-b-4 border-dashed" />
+
+                    <div className="bg-theme-highlight relative w-full" style={{ width: `${flightPlanProgress}%` }}>
+                      {!!flightPlanProgress && (
+                        <IconPlane
+                          className="text-theme-highlight absolute right-0 -translate-y-1/2 translate-x-1/2 fill-current"
+                          size={50}
+                          strokeLinejoin="miter"
+                        />
+                      )}
                     </div>
+                  </div>
+                  <p
+                    className={`font-body text-right ${Math.round(flightPlanProgress) >= 98 ? 'text-theme-highlight' : 'text-theme-text'}`}
+                  >
+                    {schedInParsed}
+                  </p>
+                </div>
+              </div>
+              <div>
+                <div className="mb-4 flex flex-row justify-around">
+                  <InformationEntry title={t('Dashboard.YourFlight.Alternate')} info={altIcao ?? 'NONE'} />
+                  <div className="bg-theme-accent mx-4 my-auto h-8 w-1" />
+                  <InformationEntry
+                    title={t('Dashboard.YourFlight.CompanyRoute')}
+                    info={departingIata + arrivingIata}
+                  />
+                  <div className="bg-theme-accent mx-4 my-auto h-8 w-1" />
+                  <InformationEntry title={t('Dashboard.YourFlight.ZFW')} info={estimatedZfw} />
                 </div>
+                <div className="bg-theme-accent my-auto h-0.5 w-full" />
+                <div className="mt-4 flex flex-row justify-around">
+                  <InformationEntry title={t('Dashboard.YourFlight.CostIndex')} info={costInd} />
+                  <div className="bg-theme-accent mx-4 my-auto h-8 w-1" />
+                  <InformationEntry title={t('Dashboard.YourFlight.AverageWind')} info={avgWind} />
+                  <div className="bg-theme-accent mx-4 my-auto h-8 w-1" />
+                  <InformationEntry title={t('Dashboard.YourFlight.CruiseAlt')} info={crzAlt} />
+                </div>
+              </div>
+              <div>
+                <h5 className="mb-2 text-2xl font-bold">{t('Dashboard.YourFlight.Route')}</h5>
+                <ScrollableContainer height={15}>
+                  <p className="font-mono text-2xl">
+                    <span className="text-theme-highlight text-2xl">
+                      {departingAirport}/{departingRunway}
```

Comment:
Really lost at trying to decipher what the linter thinks its doing here... 

Substantially less readable imo.